Core/test/qa-functional/src/org/sleuthkit/autopsy/ingest/IngestFileFiltersTest.java

```diff
@@ -156,6 +156,34 @@ public void testBasicDir() {
         }
     }
     
+    public void testExtAndDirWithOneRule() {
+        HashMap<String, Rule> rules = new HashMap<>();
+        rules.put("Rule", new Rule("testExtAndDirWithOneRule", new Rule.ExtensionCondition("jpg"), new MetaTypeCondition(MetaTypeCondition.Type.FILES), new ParentPathCondition("dir1"), null, null, null));
+        //Build the filter that ignore unallocated space and with one rule
+        FilesSet Files_Ext_Dirs_Filter = new FilesSet("Filter", "Filter to find all jpg files in dir1.", false, true, rules);
```

Comment:
Please follow Java code naming conventions - use Camel-case with no initial capital letter for local variables.

Also, I don't see that the specification for the test calls for ignoring unallocated space. Maybe it is implied in the rest of the specification? I don't know enough about the test image to know, so please clarify.